config/routes.rb

```diff
@@ -29,6 +29,13 @@
 
         put 'release_tags'
         patch 'release_tags'
+
+        resources :versions, only: [], param: :version do
+          member do
+            match 'withdraw', via: %i[put patch], defaults: { withdrawn: true }, to: 'versions#withdraw'
+            match 'restore', via: %i[put patch], defaults: { withdrawn: false }, to: 'versions#withdraw'
+          end
+        end
```

Comment:
Is it worth considering a resource-oriented approach here, where PUTs to `/v1/purls/:druid/versions/:version` will restore and DELETEs to the same URL will withdraw?

Reply:
It isn't really a delete, in that the resource isn't destroyed, it's visibility is just changed. In addition, restoring isn't the only update that one might want to make to a version.

I'm not strongly opposed to the change, but not convinced it is a great fit.

Reply:
Let's hold off on that change for now, then